packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -1625,16 +1625,20 @@ export class StripeHelper extends StripeHelperBase {
     );
   }
 
-  getPaymentProvider(customer: Stripe.Customer) {
-    const subscription = customer.subscriptions?.data.find((sub) =>
-      ACTIVE_SUBSCRIPTION_STATUSES.includes(sub.status)
-    );
-    if (subscription) {
-      return subscription.collection_method === 'send_invoice'
-        ? 'paypal'
-        : 'stripe';
+  getPaymentProvider(
+    customer: Stripe.Customer
+  ): 'not_chosen' | 'paypal' | 'stripe' {
+    const defaultPaymentMethod =
+      customer.invoice_settings.default_payment_method;
+    const paypalPaymentMethodId = customer.metadata.paypalAgreementId;
+
+    if (defaultPaymentMethod === null && paypalPaymentMethodId !== undefined) {
```

Comment:
Any reason we need to check strictly against `null` and `undefined` for these two checks rather than to check truthy/falsy? I always get a little nervous when checking external values against `null` or `undefined` explicitly since they can often be used interchangeably by external authors and can change unexpectedly.